src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentPageDefault.component.js

```diff
@@ -50,6 +51,7 @@ export const EnrollmentPageDefaultPlain = ({
                 <WidgetStagesAndEvents stages={program.stages} />
             </div>
             <div className={classes.rightColumn}>
+                <WidgetError error={widgetEffects?.errors} />
```

Comment:
I would suggest we hide this if there are no error effects at the moment. `widgetEffects?.errors && <WidgetError..`

Reply:
So remove the hiding-logic from inside the component? Or do we still keep that on as a fallback if the widget is used elsewhere?

Reply:
Lets keep your suggested logic (discussed in meeting)